pkg/brew/radicle-tools.rb

```diff
@@ -22,7 +21,8 @@ def install
     bin.install "rad-checkout"
     bin.install "rad-untrack"
     bin.install "git-remote-rad"
-    man1.install "radicle-tools.1.gz"
+    man1.install "rad.1.gz"
+    man1.install "rad-checkout.1.gz"
```

Comment:
We have a chicken and egg problem here.  Merging this hunk is going to break current `brew install ...` because the `url` above points to a tarball that doesn't have those files yet.  OTOH, the tarball having those files doesn't exist yet because it's produced when a git tag is pushed.  So: This hunk would have to land in the commit tagged `vx.x.x`.